### ISSUE TITLE
Default adaptive opacity settings

### ISSUE THREAD

**Issue by fbruetting** [micheleg/dash-to-dock#775]
I use Gnome 3.28 and the Adapta theme. Sadly, the colors of the top bar and DTD are not in sync and thus a border is always visible. I have found out, that the correct (manual) min/max opacity values are 55/100 %. Could you please fix this?

I don’t know if Adapta modifies those opacity values of Gnome Shell, or if these are the default values…

**Comment by micheleg** [comment]
It might be due to some additional trasnparency/opacity inthe theme you are using. Can you try with the default theme?

**Comment by fbruetting** [comment]
Not better with Adwaita and default transparency settings:

![bildschirmfoto von 2018-07-29 18-55-12](https://user-images.githubusercontent.com/10520767/43368705-c99b9328-9361-11e8-9c43-e4eeaa7b8988.png)
![bildschirmfoto von 2018-07-29 18-55-24](https://user-images.githubusercontent.com/10520767/43368706-c9c4600a-9361-11e8-8639-a7530e453f1e.png)

**Comment by micheleg** [comment]
Mmm, that is actually the intended behaviour I believe.

**Comment by fbruetting** [comment]
Shouldn't the intended behaviour of "adaptive" be, to adapt to the header bar's style?


Am 29.07.2018 19:34 schrieb Michele Gaio <notifications@github.com>:

Mmm, that is actually the intended behaviour I believe.

**Comment by micheleg** [comment]
In total honesty, i don't have an opinion on what's the "intended behaviour" because I do not use this feature (I'm sill on 3.22 ...). The dock is not even the same colour as the topbar (dark gray vs black), so there's no attempt to merge the two. The only attempt is to not have the top bar transparent and the dock solid, as it look "objectively" ugly. 

Obviously these are preferences, and we could change their default in the future. Hhowever, I believe the set of available settings already allow you to customize the dock as per your preferences,  am I right?

**Comment by fbruetting** [comment]
Yeah, the color mismatch is another issue. However, with the Adapta theme it fits exactly (maybe Adapta writes that color to DTD?).

If you provide a method for adapting to the top bar (which is great!), then it would be cool if you could also read color and opacity settings. I had to find out via GIMP and color probing that Gnome Shell uses 55% and 100% opacity settings. It’s not a great idea to give users wrong color and opacity settings, and let every one of them find out what opacity really should be set, if you ask me. Would be cool if you could somehow implement reading those values! :)
